frontend/packages/data-portal/app/components/MLChallenge/MainContent.tsx

```diff
@@ -300,7 +304,22 @@ export function MainContent() {
           </div>
         </div>
       </Section>
-      {/* TODO: Contact Banner */}
+
+      <Section className="font-semibold py-16" color="primary100">
+        <h2 className="text-sds-header-xl leading-sds-header-xl">
+          {t('contact')}
+        </h2>
+
+        <div className="flex justify-center gap-sds-xxl">
+          <div className="max-w-[612px]">
```

Comment:
why this max-width?

Reply:
why not just set a min-width or `flex-shrink-0` on the icon and let the rest auto-compute?

Reply:
ooh excellent point

Reply:
oh good question - flex-shrink-0 would be my preference, but really either is ok right now. 
When we get to mobile, the icon may have to shrink & stack at some breakpoint, but IDK.